### DISCUSSION TITLE
Rest and SDK exclude specific fields

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [directus/directus#23710] [Feature Requests]
### Summary

So when i want to fetch all fields except one specific, it would be nice to be able to "filter" them out.

### Basic Example

```
const policies = await directus.request<DirectusPolicies[]>(readPolicies({
			fields: ['*', "permissions.*", "roles.*", "-users"]
		}));
```

In this example i would fetch all policies but without the users ids

### Motivation

Granular fetch specific fields

### Detailed Design

Would be nice to exclude these fields either with: ```!```or ```-```

### Must Have:
- Ability to filter/exclude specific fields

### Should Have:
- Ability to filter/exclude specific fields in deep queries

### Drawbacks

Higher implementation and testing costs?

### Alternatives

Otherwise we would have to select the fields that we want. But sometimes the other way round is easier

### Adoption Strategy

No breaking changes? Unless fields can have "-" or "!" in the name

**Comment by il-jakob** [comment]
I'm also interested in this feature. Using `!` or `-` or really any disallowed character in the field name seems like a good idea. Or being able to set an "_omit" and name field you don't want.

**Comment by virgo-fondasys** [comment]
Any update for this one ??
